clients/java/src/main/java/io/camunda/zeebe/client/api/command/CommandWithTenantStep.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.client.api.command;
+
+import io.camunda.zeebe.client.api.ExperimentalApi;
+
+public interface CommandWithTenantStep<T> {
+
+  /**
+   * This identifier is used for entities that are created before multi-tenancy is enabled in the
+   * Zeebe cluster. After enabling multi-tenancy, these entities can still be interacted by using
+   * this identifier explicitly.
+   */
+  String DEFAULT_TENANT_IDENTIFIER = "[default]";
+
+  /**
+   * This identifier is used for entities that are shared among all tenants. For example, a deployed
+   * process can be shared among all tenants such that all tenants can create their own instances of
+   * the shared process definition. Process instances are always owned by a single tenant.
+   */
+  String SHARED_TENANT_IDENTIFIER = "[shared]";
```

Comment:
@korthout in the ZPA kickoff, the proposal was to use angle brackets (`<>`). This was also communicated to the other C8 teams. Using square brackets now might cause some confusion.

We can definitely discuss the change and update it if necessary, but we should communicate this change to the other teams as well.

Reply:
@koevskinikola Ah that's just a misunderstanding from my side. Let's adjust our code to fit what was discussed

Reply:
Anyways, we need to remove shared, because it's not supported in the first iteration

Reply:
To cover these, I've opened:
- #13726 
- #13746 
- #13747